.buildkite/pipeline.yml

```diff
@@ -263,6 +263,9 @@ steps:
       - reports/test-results/android-test-results.xml
     env:
       JEST_JUNIT_OUTPUT_FILE: reports/test-results/android-test-results.xml
+    notify:
+      - github_commit_status:
+          context: Test Android on Device - Canaries
```

Comment:
You can see it in action in the checks on this PR:

![image](https://github.com/wordpress-mobile/gutenberg-mobile/assets/1218433/f4daae21-b742-4017-b334-22b6c26f1946)